src/include/com/amazonaws/kinesis/video/webrtcclient/Stats.h

```diff
@@ -224,12 +224,38 @@ typedef struct {
     UINT32 selectedCandidatePairChanges; //!< The number of times that the selected candidate pair of this transport has changed
 } RtcTransportStats, *PRtcTransportStats;
 
+typedef char DOMString[MAX_STATS_STRING_LENGTH + 1];
```

Comment:
char -> CHAR. Also, think of changing to a more generic name perhaps as this can be used by other stats

Reply:
`DOMString` type is a part of webrtc spec, decided why not just use it

Reply:
It's a good idea in my opinion. The comment is mostly about using the internal type CHAR